Eatery/AppDelegate.swift

```diff
@@ -24,9 +24,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         Hero.shared.containerColor = .white
 
         // Set up view controllers
-        
-        eateryTabBarController = EateryTabBarController()
-        window?.rootViewController = eateryTabBarController
+        if UserDefaults.standard.bool(forKey: "hasOnboarded") {
```

Comment:
Should the "significantEvents" user defaults key be used here to determine whether Eatery has already been installed?

In essence, should we be presenting the Onboarding screen to users that previously had Eatery installed?

Reply:
I think we should be